components/webgpu/lib.rs

```diff
@@ -81,6 +81,27 @@ pub enum WebGPURequest {
     ),
     UnmapBuffer(WebGPUBuffer),
     DestroyBuffer(WebGPUBuffer),
+    CreateCommandEncoder(
+        IpcSender<WebGPUCommandEncoder>,
+        WebGPUDevice,
+        // TODO(zakorgy): Serialize CommandEncoderDescriptor in wgpu-core
+        // wgpu::command::CommandEncoderDescriptor,
+        wgpu::id::CommandEncoderId,
+    ),
+    CopyBufferToBuffer(
+        wgpu::id::CommandEncoderId,
```

Comment:
These should have named fields. In general, prefer the named fields for enums, unless there are only 1 or 2 values with distinctly obvious meaning

Reply:
I will create another pull request addressing this by create `parameter` structs with named fields.